README.md

```
@@ -1,56 +1,114 @@
-<h1 align="center">📸 우리만의 사진 저장소, 네키</h1>
+<img width="1920" alt="Frame 2087328876" src="https://github.com/user-attachments/assets/76eab410-423b-4442-ba36-3daead58e4a9" />
 
-<img width="2069" height="1479" alt="image 73" src="https://github.com/user-attachments/assets/33c8fdaa-559e-4d0f-a351-ea71cead6868" />
+# <img width="200" alt="Frame 2087328927" src="https://github.com/user-attachments/assets/2c0d2cf9-742a-41ad-807a-f2feabcec053" />
 
+> “Neki, 네컷의 순간이 이어지는 곳” 
 
-## Foldering
+###  네컷사진의 시작부터 보관까지, 당신의 기록을 완성하는 서비스 
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)